modules/admin_manual/pages/enterprise/firewall/file_firewall.adoc

```diff
@@ -85,6 +85,20 @@ The full list of conditions is (File mimetype upload):
 
 NOTE: {supported-mimetype-list-url}[The complete list of available mimetypes] that ownCloud supports is available the source of ownCloud core.
```

Comment:
I know that this is not part of your table, but just highlighting a possible improvement that could be catched:

`NOTE: {supported-mimetype-list-url}[The complete list of available mimetypes] that ownCloud supports is available the source of ownCloud core.`
-->
`NOTE: The complete list of available mimetypes that ownCloud supports is available in the {supported-mimetype-list-url}[source of ownCloud core.]`

Reply:
I won't change something @settermjd just worked in the document. Sorry but that's rude :)